client/setup.py

```diff
@@ -1,12 +1,12 @@
 from setuptools import setup, find_packages
 
 setup(
-    name='s1_client',
-    version='0.1.0',
+    name='s1_client_test',
```

Comment:
Change to ciaobjectstore

Reply:
change name to ciastore right?
that measn i have to change the folder name s1_client to ciastore

Reply:
yeah. If ciaos is available then go with ciaos